scripts/local/docker-compose.yml

```diff
@@ -0,0 +1,148 @@
+# This is the docker-compose script for local development.
+# For the latest version of the docker images see:
+# https://github.com/Azure/pcs-cli/blob/master/remotemonitoring/single-vm/docker-compose.dotnet.yml
+#
+# Usage:
+# - install Docker Compose, see https://docs.docker.com/compose/install
+# - open a console and `cd` into the folder that contains this file
+# - note how this file references many environment variables, either set
+#   them in the .env file or edit this file adding the values here
+# - run `docker-compose up`
+# - open the browser at http://127.0.0.1:10443
+#
+# For more information see
+# https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet
+
+version: "3"
+
+services:
+  reverseproxy:
+    image: azureiotpcs/remote-monitoring-nginx:testing
+    ports:
+      - "80:80"
+      - "443:443"
+    depends_on:
+      - webui
+      - auth
+      - iothubmanager
+      - devicesimulation
+      - telemetry
+      - config
+    volumes:
+    # Required volume for app certs
```

Comment:
might want to make sure they know that they need to mount their local drive so docker can access the volume.

Reply:
Interesting, I didn't mount a local drive but it worked for me when deploying locally. Or maybe I did and I just didn't know :).